environment-dev.yml

```diff
@@ -40,9 +40,9 @@ dependencies:
   - pandas-stubs
   - fastparquet
   - pip:
-      # TODO(https://github.com/modin-project/modin/issues/5171): uncap xgboost
-      # when we can run tests on 1.7.0 without getting stuck.
-      - xgboost>=1.6.0,<1.7.0
```

Comment:
I think we should not remove the lower bound.

Reply:
agreed, let's not lose pinnings

Reply:
Now using xgboost>=1.7.1,<2.0.0 everywhere